src/main/java/de/hysky/skyblocker/annotations/Init.java

```diff
@@ -0,0 +1,8 @@
+package de.hysky.skyblocker.annotations;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Target;
+
+@Target(ElementType.METHOD)
+public @interface Init {
```

Comment:
I would rename it to SkyblockerInit to make it a little less vague. Also javadoc doesn't hurt :p 
but go to sleep for now you tired fool